server/tests/io/fixtures/belga_remote_newsml_1_2.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="ISO-8859-15" standalone="yes"?>
+<NewsML>
+    <Catalog Href="http://www.belga.be/dtd/BelgaCatalog.xml"/>
+    <NewsEnvelope>
+        <DateAndTime>20190603T160219</DateAndTime>
+        <NewsService FormalName=""/>
+        <NewsProduct FormalName=""/>
+    </NewsEnvelope>
+    <NewsItem>
+        <Identification>
+            <NewsIdentifier>
+                <ProviderId>belga.be</ProviderId>
+                <DateId>20190603T160217</DateId>
+                <NewsItemId>0</NewsItemId>
+                <RevisionId Update="N" PreviousRevision="0">0</RevisionId>
+                <PublicIdentifier>urn:newsml:www.belga.be</PublicIdentifier>
+            </NewsIdentifier>
+        </Identification>
+        <NewsManagement>
+            <NewsItemType FormalName="NEWS"/>
+            <FirstCreated>20190603T160217</FirstCreated>
+            <ThisRevisionCreated>20190603T160217</ThisRevisionCreated>
+            <Status FormalName="USABLE"/>
+        </NewsManagement>
+        <NewsComponent Duid="0" xml:lang="nl">
+            <NewsLines>
+                <HeadLine>Knooppunt Schijnpoort in Antwerpen hele zomervakantie afgesloten</HeadLine>
+            </NewsLines>
+            <AdministrativeMetadata/>
+            <DescriptiveMetadata>
+                <Genre FormalName=""/>
+            </DescriptiveMetadata>
+            <NewsComponent Duid="0" xml:lang="nl">
```

Comment:
I wonder if all `belga remote` files have `Duid="0"`, if so, we must adjust a logic. Please, clarify this moment.

Reply:
Hi, @ride90 
IMO, In `Belga remote` we should use `dateid`(instead of `Duid`) for `guid`. and I will split the `Belga remote` to another feed parser. Is is ok?

Reply:
Do you think `dateid` is unique? For me, it looks like it's not unique. Can you clarify why `Duid` is 0 with Stefaan?

P.S. If custom logic is needed let's move Belga Remote to a standalone parser.